Luke/Import doc changes (#3032)
FusionAuth/fusionauth-site: astro/src/content/docs/lifecycle/migrate-users/index.mdx

### Bulk Migration

> Bulk migration lets you import users at one point in time, making for a clean cutover. You can import password hashes, including hashes performed with a custom, non-standard algorithm, making a migration seamless to your users. You use the [User Import API](/docs/apis/users#import-users) to perform this. You can also [import refresh tokens](/docs/apis/users#import-refresh-tokens), which helps if you want your users to be able to refresh access tokens transparently. When using the [User Import API](/docs/apis/users#import-users) it is suggested to import in batches of less than 10,000 users per request. After completing a migration it is suggested that a [reindex of the Elasticsearch database](/docs/lifecycle/manage-users/search/search#reindexing-elasticsearch) is completed as well. 

Reviewer: ```suggestion
Bulk migration lets you import users at one point in time, making for a clean cutover. You can import password hashes, including hashes performed with a custom, non-standard algorithm, making a migration seamless to your users. You use the [User Import API](/docs/apis/users#import-users) to perform this. You can also [import refresh tokens](/docs/apis/users#import-refresh-tokens), which helps if you want your users to be able to refresh access tokens transparently. When using the [User Import API](/docs/apis/users#import-users) you should import with batches of less than 10,000 users per request. After completing a migration you should [reindex of the Elasticsearch database](/docs/lifecycle/manage-users/search/search#reindexing-elasticsearch) as well. 
```
